public/video-ui/src/components/ScheduledLaunch/ScheduledLaunch.js

```diff
@@ -130,6 +131,18 @@ export default class ScheduledLaunch extends React.Component {
     this.setState({ showDatePicker: false, [propertyName]: null });
   };
 
+  getNoScheduleReason = () => {
+    if (!this.props.video.title) {
+      return 'You must add a title before scheduling';
+    }
+
+    if (this.props.hasPublishedVideoPageUsages()) {
```

Comment:
can we add `hasPublishedVideoPageUsages` to the `propTypes` [above](https://github.com/guardian/media-atom-maker/pull/731/files#diff-07b21fff0b6992387e6c2e1c96395ef5R17)?